Lib/test/test_argparse.py

```diff
@@ -1938,7 +1938,23 @@ def test_required_subparsers_no_destination_error(self):
         parser = ErrorRaisingArgumentParser()
         subparsers = parser.add_subparsers()
         subparsers.add_parser('run')
-        self.assertArgumentParserError(parser.parse_args, ())
+        with self.assertRaises(ArgumentParserError) as excinfo:
+            parser.parse_args(())
+        self.assertRegex(
+            excinfo.exception.stderr,
+            'error: the following arguments are required: {run}\n$'
```

Comment:
The braces look funny here.

Reply:
yeah they look a little weird with only one subparser (not multiple choice) -- I'll update the test to be a little more realistic.